app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class TargetsController < ApiController
+      def create
+        target.save!
```

Comment:
This should be something like `current_user.targets.create!(...)`

Reply:
I think @sebastiancaraballo's suggestion solves the same thing in only one line 🙂

Reply:
So then you can do `@target = current_user.targets.create!(...)`